### ISSUE TITLE
SSD-VGG16, Retrain on Open Images Dataset

### ISSUE THREAD

**Issue by hoangcuongbk80** [qfgaohao/pytorch-ssd#138]
Hi,

I have trained SSD with backbone VGG16 for Gun dataset. The error I got for different pre-trained models:
 
vgg16_reducedfc:
`python train_ssd.py --dataset_type open_images --datasets /home/aass/Hoang-Cuong/ekobot/data --net "vgg16-ssd" --base_net models/vgg16_reducedfc.pth  --batch_size 24 --num_epochs 150 --scheduler cosine --lr 0.0012 --t_max 150 --validation_epochs 5`

![reducedfc](https://user-images.githubusercontent.com/10355081/93000012-ac357100-f525-11ea-9042-a309b3ed5732.png)

vgg16-ssd-mp-0_7726:
`python train_ssd.py --dataset_type open_images --datasets /home/aass/Hoang-Cuong/ekobot/data --net vgg16-ssd --base_net models/vgg16-ssd-mp-0_7726.pth  --batch_size 24 --num_epochs 150 --scheduler cosine --lr 0.0012 --t_max 150 --validation_epochs 5`

![7726](https://user-images.githubusercontent.com/10355081/93000014-afc8f800-f525-11ea-9d84-0a4808a3aa17.png)

Thanks

**Comment by voidzer0-development** [comment]
The state dict of the pretrained model you are trying to load and the one defined in vgg_ssd.py seem to be different. Can you show the runtimeError of the second screenshot?
